src/ol/source/wms.jsdoc

```diff
@@ -1,3 +0,0 @@
-/**
- * @namespace ol.source.wms
```

Comment:
Have you checked if `ol.source.WMSServerType` is properly listed in the API docs after this change?